backend/.env.example

```diff
@@ -15,3 +15,7 @@ BRANCH=""
 GITHUB_EMAIL=""
 #Add your github access token
 TOKEN=""
+# Add your google client id
+GOOGLE_CLIENT_ID=""
+# Add your google client scret
+GOOGLE_CLIENT_SECRET=""
```

Comment:
@sadath2001 when these values are not used anywhere in the backend why add them in the env variable?

Reply:
hi @AlfiyaSiddique , ill remove them right away, i added them while testing, forgot to remove them.
Any additional changes needed? let me know so that i can make them all at  once and submit the review.